doc-content/drools-docs/src/main/asciidoc/DecisionEngine/stateless-kie-sessions-globals-con.adoc

```diff
@@ -0,0 +1,49 @@
+[id='stateless-kie-sessions-globals-con_{context}']
+
+= Global variables in stateless KIE sessions
+
+The `StatelessKieSession` object supports global variables (globals) that you can configure to be resolved as session-scoped globals, delegate globals, or execution-scoped globals.
+
+* *Session-scoped globals:* For session-scoped globals, you can use the method `getGlobals()` to return a `Globals` instance that provides access to the KIE session globals. These globals are shared for all execution calls. Use caution with mutable globals because execution calls can be executing simultaneously in different threads.
```

Comment:
"shared" or "used"?

Reply:
Good call, "used for all" = "shared", so kind of a redundancy. Changing to "used".